RadixWallet/Features/GatewaySettingsFeature/Children/AddNewGateway/AddNewGateway+View.swift

```diff
@@ -63,11 +74,14 @@ extension AddNewGateway {
 								to: $focusedField
 							)
 						)
+						.padding(.top, .small3)
 						.textInputAutocapitalization(.never)
 						.keyboardType(.URL)
 						.autocorrectionDisabled()
 					}
-					.padding([.bottom, .horizontal], .medium1)
+					.padding(.top, .medium3)
+					.padding(.bottom, .medium1)
+					.padding(.horizontal, .large2)
```

Comment:
Very tiny style comment: we usually put them in this order: top, horizontal (or leading and trailing), bottom